localisation/00_wc_economy.csv

```diff
@@ -0,0 +1,19 @@
+#CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
+borrow_money;Borrow 300�;;Leiht Euch 300�;;;;;;;;;;x;
+borrow_money_desc;You can borrow 300� from some rich ruler or moneylenders.;;Ihr k�nnt Euch 300� von einem reichen Herrscher oder Geldverleiher leihen.;;;;;;;;;;x;
+repay_money;Repay 360� Loan;;Zahlt 360� Darlehen zur�ck;;;;;;;;;;;x
+repay_money_desc;You repay the gold you owe.;;Ihr Zahlt das Darlehen zur�ck, das Ihr schuldig seid.;;;;;;;;;;;x
+debt_360_gold;Owes 360�;;360� Darlehen;;;;;;;;;;;x
+interest_payment_360_gold;Receives Interest Payments;;Erh�lt Einkommen aus Zinsen;;;;;;;;;;;x
+EVTDESC_WCECO_100;You ask your servants to look for moneylenders who can borrow gold for your goals.\n\nWithin the day, they return with news.;;Ihr bittet Eure Diener, nach Geldverleihern zu suchen, die Euch Gold f�r Eure Ziele leihen k�nnen.\n\nIm Laufe des Tages kehren sie mit Neuigkeiten zur�ck.;;;;;;;;;;x;
+EVTOPTA_WCECO_100;[target_moneylender.GetFirstName] you say... deliver [target_moneylender.GetHerHim] my letter!;;[target_moneylender.GetFirstName] sagt ihr... �berbringt [target_moneylender.Getihmihr] meinen Brief!;;;;;;;;;;x;
+EVTOPTB_WCECO_100;The local community is giving loans... interesting...;;Die hiesige Gemeinde gew�hrt Darlehen... interessant...;;;;;;;;;;x;
+EVTDESC_WCECO_110;[Root.PrimaryTitle.GetFOA],\nlegends are told about your wealth. May you allow me to borrow a part of your gold. We will return all the gold to the last coin and more.\n\nSigned,\n[target_borrower.GetTitledName];;[Root.PrimaryTitle.GetFOA],\nes werden Legenden �ber Euren Reichtum verbreitet. D�rfte ich mir einen Teil Eures Goldes leihen? Wir werden das Gold bis zur letzten M�nze zur�ckzahlen und mehr.\n\nGezeichnet,\n[target_borrowrower.GetTitledName];;;;;;;;;;x;
+EVTDESC_WCECO_115;[Root.PrimaryTitle.GetFOA],\nit will be my honor to be your sponsor. You can pay off your debt whenever you want.\n\nSigned,\n[target_moneylender.GetTitledName];;[Root.PrimaryTitle.GetFOA],\nes w�re mir eine Ehre, Euer Sponsor zu sein. Ihr k�nnt Eure Schulden begleichen, wann immer Ihr wollt.\n\nGezeichnet,\n[target_moneylender.GetTitledName];;;;;;;;;;x;
+EVTDESC_WCECO_120;[Root.PrimaryTitle.GetFOA],\nto my great regret, I am not able to lend you money.\n\nSigned,\n[target_moneylender.GetTitledName];;[Root.PrimaryTitle.GetFOA],\nzu meinem gro�en Bedauern bin ich nicht in der Lage, Ihnen Geld zu leihen.\n\nGezeichnet,\n[target_moneylender.GetTitledName];;;;;;;;;;x;
+EVTDESC_WCECO_150;It has been a long time since you heard any news from [target_moneylender.GetTitledFirstName].\n\nYou have no idea what happened, but at this point it looks like we can consider this transaction to be failed.;;Es ist schon lange her, dass Ihr Neuigkeiten von [target_moneylender.GetTitledFirstName] geh�rt habt.\n\nIhr habt keine Ahnung, was passiert ist, aber an diesem Punkt sieht es so aus, als ob wir diese Transaktion als gescheitert betrachten k�nnen.;;;;;;;;;;x;
```

Comment:
```suggestion
EVTDESC_WCECO_150;It has been a long time since you heard any news from [target_moneylender.GetTitledFirstName].\n\nYou have no idea what has happened, but at this point, it looks like we can consider this transaction void.;;Es ist schon lange her, dass Ihr Neuigkeiten von [target_moneylender.GetTitledFirstName] geh�rt habt.\n\nIhr habt keine Ahnung, was passiert ist, aber an diesem Punkt sieht es so aus, als ob wir diese Transaktion als gescheitert betrachten k�nnen.;;;;;;;;;;x;
```